components/atoms/NotificationsCard/notification-card.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { FaRegSmile, FaUserCircle } from "react-icons/fa";
+
+interface NotificationCard {
+  type: "reaction" | "follow" | "collaboration";
+  message: string;
+}
+const NotificationCard = ({ type, message }: NotificationCard) => {
```

Comment:
Suggest this if the purpose is just rotating icons and not any custom design:

```
const NotificationCard = ({ type, message }: NotificationCard) => {
  const Icons = {
    reaction: FaRegSmile,
    follow: FaUserCircle,
    collaboration: FaUserCircle
  };
  
  const Icon = Icons[type];

  return (
    <div className="flex items-center gap-3 p-2 transition cursor-pointer rounded-xl group hover:text-sauced-orange hover:bg-sauced-light">
      <Icon className="text-3xl shrink-0"/>
      <p className="text-sm">{message}</p>
    </div>
  );
}
```

Reply:
Thanks!